### PR TITLE
[11.0][MIG] delivery_carrier_label_postlogistics

### PR DESCRIPTION
**Pull request by grindtildeath**
Based on #128 
Depends on : 
 - [x] #156 
 - [x] https://github.com/OCA/server-tools/pull/1261

### CODE REVIEW

File: delivery_carrier_label_postlogistics/models/delivery.py [hunk 1]
```diff
@@ -109,18 +109,20 @@ class DeliveryCarrierOption(models.Model):
...
-            (be it from cache or context) """
+    @api.depends('carrier_id.allowed_tmpl_options_ids')
+    def _compute_allowed_tmpl_options_ids(self):
+        """ Gets the available template options from related delivery.carrier
+            (be it from cache or context)."""
         for option in self:
-            domain = self.env.context.get('default_tmpl_option_id_domain')
-            option.tmpl_option_id_domain = (
-                option.carrier_id.allowed_options_domain or domain or '[]')
+            domain = self.env.context.get('default_allowed_tmpl_options_ids')
```
Comment (ghost) [inline delivery_carrier_label_postlogistics/models/delivery.py]: doesn't appear to me that `domain` is the correct name for the content of it now

File: delivery_carrier_label_postlogistics/models/delivery.py [hunk 2]
```diff
@@ -109,18 +109,20 @@ class DeliveryCarrierOption(models.Model):
...
+            option.allowed_tmpl_options_ids = (
+                option.carrier_id.allowed_tmpl_options_ids or domain or '[]')
```
Comment (ghost) [inline delivery_carrier_label_postlogistics/models/delivery.py]: `or '[]'` seems quite dangerous to use here, since you're writing to `Many2many` :smile:

**Review by tschanzt — COMMENTED** [review]
There is a lot of commented xml around. I think you should add a comment why it is still needed or it should be dropped.

**Comment by asaunier** [comment]
Perhaps you could squash the old consecutive commits of the same authors?

**Comment by grindtildeath** [comment]
@asaunier I will do it as soon as https://github.com/OCA/delivery-carrier/pull/128 is merged, but I prefer keeping it so if more is to be changed.

**Comment by simahawk** [comment]
https://github.com/OCA/server-tools/pull/1261 merged

Build red

**Comment by grindtildeath** [comment]
@simahawk @yvaucher @tschanzt @sebalix @asaunier 
Can you guys please pass byy for a review and get this old PR merged ?

File: oca_dependencies.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# list the OCA project dependencies, one per line
+# add a github url if you need a forked version
+partner-contact
+server-tools
+stock-logistics-workflow
+webkit-tools
+web
```
Comment (sebalix) [inline oca_dependencies.txt:7]: We only need `server-tools` here (the module depends on `configuration_helper` from OCA/server-tools and `base_delivery_carrier_label` depending on `delivery` only which is a standard one).

**Review by sebalix — CHANGES_REQUESTED** [review]

**Review by tschanzt — APPROVED** [review]

**Review by asaunier — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

File: oca_dependencies.txt (hunk 3 above)
Comment (pedrobaeza) [reply oca_dependencies.txt:7]: Please attend this comment for not making slower this CI

**Comment by sebalix** [comment]
@grindtildeath can you cherry-pick the commit from https://github.com/OCA/delivery-carrier/pull/191 too?

**Pull request merged**